### PR TITLE
Heatmap Tracking App

### PR DESCRIPTION
**Pull request by bolt12**
Deleted similar projects and added one that somewhat merges them alltogether.

Added heatpmap_tracking_app project proposal.

### CODE REVIEW

**Comment by hairychris** [comment]
I think this change looks good, let's get a 3rd opinion as this does delete other peoples work.

**Comment by bolt12** [comment]
Okay!

**Comment by Sanguine76** [comment]
Hello gentlemen, anyone here with experience in phone GPS history? This is quite an essential component of what we are dealing with… Every person has a cell phone. We can backtrack in history to see who was exposed… I have much more to say on this topic. Who is ready to crack it open?

File: software/proposals/heatpmap_app.md [hunk 1]
```diff
@@ -1,50 +0,0 @@
...
-
-## 💪 Impact of Potential Solution
-
-## 🙌 Criteria for Success
-
-## 😅 Known Issues
-
-## ✅ "I've read this" section
-
-| Name | Status |
```
Comment (Sanguine76) [inline software/proposals/heatpmap_app.md:48]: Phil Huerter Read. Like it!! Want to expand this topic

**Comment by siddharthbharthulwar** [comment]
At least for phones regularly connected to cellular service or internet, Google Maps already keeps track of location history for weeks. We can leverage this location history as well as newly recorded GPS history to find prior possible exposures as well.

File: software/proposals/tracking_app.md [hunk 2]
```diff
@@ -1,54 +0,0 @@
...
-
-## 🙌 Criteria for Success
-
-## 😅 Known Issues
-
-## ✅ "I've read this" section
-
-| Name | Status |
-|-|-|
-|  |  Read/Not Read    |
```
Comment (Sanguine76) [inline software/proposals/tracking_app.md:54]: I have read and want to collaborate with Henrique Mota and Armando Santos

**Review by Sanguine76 — COMMENTED** [review]
Let’s talk about this. We want to color code red =  contagious... Leaves a line trail, With location history Backtracked 14 days*** that’s it! The most important part.
Yellow = symptoms but untested/confirmed
Green = healthy/no symptoms/test negative 
These are Dots on Google Map... 
moving live.
Key Feature: People Get a Geofence around them... maybe 100 ft (or more) To Avoid the track of someone found Contagious (Red)
Yes. This is public. The track will look like Red Spaghetti 🍝 and people will avoid these areas like the Plague (this is the Key feature)
We have a National phone database via FEMA emergency response Alert. I have contacted their DC office. May gain access to all Cell #’s for proximity Warning as they cross a ‘GeoFence’ contagion boundary.

**Comment by Sanguine76** [comment]
Thank you for your comment. Did you see
Mine? Is it faster to use Skack? What’s the
best way to collaborate?
I have the Whole thing figured out but need
a coder such as yourself. There’s OpenPath.cc
available for GPS phone history in the Play Store.
Let’s talk!
Phil

**Comment by siddharthbharthulwar** [comment]
Slack is probably the best way to collaborate. My slack profile name is my full name (Siddharth Bharthulwar). 

The colored map sounds like a good idea, and shouldn't be too hard to implement with existing maps frameworks. If we can figure out how to aggregate the location history for every user and store it for easy access, the rest should be simple. Everyone would have to install the app and give it location history permissions for it to be effective.

**Review by mazaira — APPROVED** [review]
I'm good with the merge. Thanks for pushing it :)

**Comment by bolt12** [comment]
Hey @Sanguine76 and @siddharthbharthulwar thanks for starting a discussion!

So I think we must focus on the core idea: 

- Have someway to collect information about the affluence in public places (supermarkets, parks, restaurants, etc.). 
- The public info must be anonym: People can only see some kind of heatmap on a google maps like interface. However private info can tell them their history.
- People can enter the day/time they felt COVID-19 symptoms and the app should notify all people that were nearby that they might have had a contact and add a mark on the heatmap.

We do not think we need anything else for Version 1.

We can deal with problems that arise from crowd-sourced apps later like false positives notification spreads.

We assume people will have their GPS/Internet/Mobile data on all the time.

I do not have experience with GPS or tracking software but I'm aware that Google Maps does offer a lot of the information that we want.

**Comment by Sanguine76** [comment]
Thank you for the review Armando how’s your day going?

Phil

**Comment by bolt12** [comment]
Going well, thanks! Let's hope the project gets approved by Project Open Air!

**Comment by Sanguine76** [comment]
How's progress?

**Pull request merged**